packages/core/directives/polyline-point.ts

```diff
@@ -33,4 +41,12 @@ export class AgmPolylinePoint implements OnChanges {
       this.positionChanged.emit(position);
     }
   }
+
+  /** @internal */
+  getFitBoundsDetails$(): Observable<FitBoundsDetails> {
+    return this.positionChanged.pipe(
+      startWith({lat: this.latitude, lng: this.longitude}),
+      map(x => ({latLng: x}))
```

Comment:
rename `x` to `position` or some other meaningful name

Reply:
Thanks, used `position`